src/Microsoft.ML.Experimental/DataLoadSave/Database/DatabaseLoader.cs

```diff
@@ -278,15 +300,30 @@ public Range(int min, int max)
             /// <summary>
             ///  The minimum index of the column, inclusive.
             /// </summary>
+            /// <remarks>
+            /// This is <c>-1</c> if the range represents a column name.
```

Comment:
Since these fields are publicly settable, this doesn't always hold true. The following code is possible:

```C#
Range r = new Range();
r.Name = "foo";
r.Min = 3;
```

Maybe we should somehow relay in the XML docs that `Name` will override `Min` and `Max`.

Reply:
Changed to:
```
/// This value is ignored if <see cref="Name" /> is not <c>null</c>.
```

and
```
/// This value, if non-<c>null</c>, overrides <see cref="Min" /> and <see cref="Max" />.
```